cmd/validate.go

```diff
@@ -27,7 +27,7 @@ var ValidateCmd = &cobra.Command{
 		}
 		defer hst.Close()
 
-		logger.Info("🔍 Validating", "path", path, "host", hst)
+		logger.Info("🔍 Validating", "path", path, hst.Type(), hst)
```

Comment:
ah...`slog`... So, the interface is... a bit odd, I reckon: `Info(message, key1, value1, key2, value2 etc)`.

so, this would make more sense I think:

```suggestion
		logger.Info("🔍 Validating", "path", path, "target", hst)
```

so, it'll use `Host.String()`. BUT, thiss would hide the type, so it'd either have to be something like:

- `fmt.Sprintf("%s:%s", hst.Type(), hst.Strin())`
- Update all `Host.String()` to do that.

WDYT?